lib/run.js

```diff
@@ -82,3 +82,19 @@ function isProfileName (profile) {
   }
   return !/[\\\/]/.test(profile);
 }
+
+/**
+ * Concat binary arguments to arguments array.
+ *
+ * @param {Array} args - arguments
+ * @param {string|Array} binArgs - binary arguments
+ * @return {Array} - arguments
+ */
+function concatBinArgs (args, binArgs) {
+  args = Array.isArray(args) && args || [];
+  if (typeof binArgs === "string" || binArgs instanceof String) {
+    binArgs = binArgs.match(/\"(?:[^"]|\\\")*\"|(?:[^"\s]|\")+/g);
```

Comment:
I guess we should probably support single quotes too, since most shells support that?